tests/rule_based_profiler/data_assistant/test_onboarding_data_assistant_happy_paths.py

```diff
@@ -391,14 +391,17 @@ def test_sql_happy_path_onboarding_data_assistant_null_column_quantiles_metric_v
 @pytest.mark.integration
 @pytest.mark.slow  # 26.57 seconds
 def test_sql_happy_path_onboarding_data_assistant_mixed_decimal_float_and_boolean_column_unique_proportion_metric_values(
-    sa,
     empty_data_context,
+    test_backends,
```

Comment:
Added test_backends so we can detect whether we are suppose to run postgres tests. I've also reordered these to be consistent with the other postgres test in this file.